### DISCUSSION TITLE
今年HoYoFair也很精彩😭

### DISCUSSION THREAD

**Discussion by Andrea0126** [share121/inter-knot#5282] [原神]
![IMG_20240921_234417](https://github.com/user-attachments/assets/5d0b6de3-08f5-4891-95bd-6ec345ed4830)
![IMG_20240921_234433](https://github.com/user-attachments/assets/d3ab2dab-a454-4e0c-bcc5-e1deb29016c4)
![IMG_20240921_234444](https://github.com/user-attachments/assets/11838318-3b5f-4a42-aa21-4a3cd7383f82)
從影片裡截的圖，最喜歡的三個部分
大家喜歡的是哪些呢

**Comment by Lun603** [comment]
截圖是真的好看啊!這次我也看得很滿足~真喜歡這種遊戲二創的氛圍!
